### PR TITLE
FAQ page stubbed out and base styles applied.

### PR DESCRIPTION
**Pull request by JoeKarlsson1**
The FAQ page, collection and template have all been hooked up and I added content on Airship. I have base styles applied, but it still needs work once we have a more consistent look and feel for the site.

### CODE REVIEW

**Comment by jaywon** [comment]
Merged pull request #30 and it caused merge conflicts with this PR. Looks like it's in the generated css file. Can you update PR plz. Thx!

**Comment by JoeKarlsson1** [comment]
I removed the styles.css from Github so it'll be a clean merge. It should be good to go now.

**Comment by jaywon** [comment]
duuuude @JoeKarlsson1 so sorry, merged newer PR request before this one and caused a conflict again I think. Can you fix again and I'll get this one in order and on master before another PR gets sent. Ping me via chat so I can act on it right away

**Comment by JoeKarlsson1** [comment]
Alright - I fixed up the merge conflicts 🎉

File: compartments/layouts/main-layout.html [hunk 1]
```diff
@@ -154,7 +154,10 @@
               <a class="black-text waves-effect waves-light" href="/faq">FAQ</a>
             </li>
             <li>
-              <a class="black-text waves-effect waves-light" href="/contact">Contact</a>
+              <a class="black-text waves-effect waves-light" href="http://localhost:9001/faq">FAQs</a>
+            </li>
+            <li>
+              <a class="black-text waves-effect waves-light" href="http://localhost:9001/contact">Contact</a>
```
Comment (jaywon) [inline compartments/layouts/main-layout.html]: Can we make the paths relative and not absolute

File: compartments/layouts/main-layout.html [hunk 2]
```diff
@@ -154,7 +154,10 @@
               <a class="black-text waves-effect waves-light" href="/faq">FAQ</a>
             </li>
             <li>
-              <a class="black-text waves-effect waves-light" href="/contact">Contact</a>
+              <a class="black-text waves-effect waves-light" href="http://localhost:9001/faq">FAQs</a>
```
Comment (jaywon) [inline compartments/layouts/main-layout.html]: Update path here also.

**Comment by JoeKarlsson1** [comment]
Updated the paths to be relative, not absolute.

**Pull request merged**